.konflux/scripts/render_snapshot.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+import datetime
+import json
+import os
+
+
+def parse_image_refs(image_refs):
+    return json.loads(image_refs)
+
+
+def validate_component(component):
+    assert (
+        component["component"] != ""
+        or component["ref"] != ""
+        or component["revision"] != ""
+        or component["repository"] != ""
```

Comment:
This must be `and`, not `or`.

```
>>> assert(True or False), "blah!"
>>> assert(True and False), "blah!"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: blah!
>>> 
```

Reply:
fixed in https://github.com/stackrox/stackrox/pull/13577/commits/c87fd5fc0f26269f5791755e6a9832aa19369f65.
Good catch - I didn't invert the condition when switching from an `if`.